client/my-sites/domains/domain-management/transfer/transfer-out/select-ips-tag.jsx

```diff
@@ -0,0 +1,227 @@
+	}
+
+	getSuggestions() {
+		return this.state.ipsTagList
+			.filter( hint => this.state.currentQuery && startsWith( hint.tag, toUpper( this.state.currentQuery ) ) )
+			.map( hint => ( { label: hint.tag + '  (' + hint.registrarName + ')' } ) );
+	}
+
+	setSuggestionsRef = ref => ( this.suggestionsRef = ref );
+
+	hideSuggestions = () => this.setState( { currentQuery: '' } );
+
+	popOverDialog = () => {
+		const { ipsTagInput, ipsTagList } = this.state;
+		let selectedRegistrar = this.getRegistrarInfo( ipsTagInput, ipsTagList );
+
+		if ( isEmpty( selectedRegistrar ) ) {
+			selectedRegistrar = { tag: toUpper( ipsTagInput ), registrarName: '', registrarUrl: '' };
+		}
+
+		this.setState( {
+			selectedRegistrar,
+			showDialog: true,
+		} );
+	}
+
+	onCloseDialog = ( action ) => {
+		if ( 'submit' === action ) {
+			this.handleSubmit();
+		}
+
+		this.setState( { showDialog: false } );
+	}
+
+	renderGoToGainingRegistrar() {
+		const { translate, gainingRegistrar: { registrarUrl } } = this.props;
+
+		return (
+			<Card
+				className="transfer-out__select-ips-tag"
+				href={ registrarUrl }
+				target="_blank"
+				rel="noopener noreferrer"
+				highlight="success"
+			>
+			{
+				registrarUrl
+				? translate( 'Success! Click here to go to your new registrar and complete the transfer.' )
+				: translate( 'Success! Contact your new registrar to complete the transfer.' )
+			}
+			</Card>
+		);
+	}
+
+	renderIpsTagSelect() {
+		const { isSaving, selectedDomainName, translate } = this.props;
+
+		return (
+			<div>
+				<p>
+					{ translate(
+						'Please enter the IPS tag of the registrar you wish to transfer ' +
+						'{{strong}}%(selectedDomainName)s{{/strong}} to.',
+						{ args: { selectedDomainName }, components: { strong: <strong /> } }
+					) }
+				</p>
+
+				<SearchCard
+					disableAutocorrect
+					onSearch={ this.handleSearchInputChange }
+					onBlur={ this.hideSuggestions }
+					onKeyDown={ this.handleKeyDown }
+					placeholder={ translate( 'Start typing an IPS tag…' ) }
+					value={ this.state.ipsTagInput }
+				/>
+				<Suggestions
+					ref={ this.setSuggestionsRef }
+					query={ this.state.currentQuery }
+					suggestions={ this.getSuggestions() }
+					suggest={ this.handleSuggestionClick }
+				/>
+				<FormButton onClick={ this.popOverDialog } disabled={ isSaving }>
+					{ translate( 'Submit' ) }
+				</FormButton>
+
+				{ this.renderDialog() }
+			</div>
+		);
+	}
+
+	renderDialog() {
+		const { selectedDomainName, translate } = this.props;
+		const { selectedRegistrar } = this.state;
+		const buttons = [
+			{ action: 'cancel', label: translate( 'Cancel' ) },
+			{ action: 'submit', label: translate( 'Yes, Submit!' ), isPrimary: true },
+		];
+
+		return (
+			<Dialog
+				baseClassName="transfer__ips_tag_set_dialog"
+				buttons={ buttons }
+				isVisible={ this.state.showDialog }
+				onClose={ this.onCloseDialog }>
+				<h1>{ translate( 'Transfer Confirmation' ) }</h1>
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Transfer confirmation required' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/uk-transfer-out)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).